.github/workflows/publish.yml

```diff
@@ -60,8 +56,9 @@ jobs:
         # GITHUB_REF has a form of `refs/tags/v1.3.0`.
         # TAG_NAME would be `v1.3.0`
         TAG_NAME=${GITHUB_REF#refs/tags/}
-        ./scripts/tag_image_and_push.sh playwright-python:localbuild playwright.azurecr.io/public/playwright-python:latest
-        ./scripts/tag_image_and_push.sh playwright-python:localbuild playwright.azurecr.io/public/playwright-python:${TAG_NAME}
-
-        ./scripts/tag_image_and_push.sh playwright-python:localbuild playwright.azurecr.io/public/playwright-python:focal
-        ./scripts/tag_image_and_push.sh playwright-python:localbuild playwright.azurecr.io/public/playwright-python:${TAG_NAME}-focal
+        docker tag playwright-python:localbuild playwright.azurecr.io/public/playwright-python:latest
```

Comment:
`docker push --all-tags` is much cleaner than using an custom script to tag and build every image

Reply:
We tried it awhile before in our main repo `microsoft/playwright` and it didn't work. Thus the script 🤷‍♂️

I wouldn't touch this for now unless there's a strong urge.

Reply:
@aslushnikov sure it would revert it